static/funding.json

```diff
@@ -0,0 +1,148 @@
+{
+    "version": "v1.0.0",
+
+    "entity": {
+        "type": "organisation",
+        "role": "owner",
+        "name": "The Matrix.org Foundation",
+        "email": "managing-director@foundation.matrix.org",
+        "description": "Neutral custodian of the Matrix community assets",
+        "webpageUrl": {
+            "url": "https://matrix.org"
+        }
+    },
+
+    "projects": [{
+        "guid": "morg-server",
+        "name": "Matrix.org homeserver",
+        "description": "The Matrix.org homeserver used for the general public to get onboarded easily on Matrix for free. Operating the server costs $240k US.",
+        "webpageUrl": {
+            "url": "https://matrix.org"
+        },
+        "repositoryUrl": {
+            "url": "https://github.com/matrix-org/matrix.org",
+            "wellKnown": "https://github.com/matrix-org/matrix.org/blob/main/.well-known/funding-manifest-urls"
+        },
+        "licenses": ["spdx:Apache-2.0"],
+        "tags": ["community", "instant-messaging", "communication", "privacy", "security", "bots", "api", "server"]
+    },{
+        "guid": "morg-spec",
+        "name": "Matrix Specification",
+        "description": "Matrix defines a set of open APIs for decentralised communication, suitable for securely publishing, persisting and subscribing to data over a global open federation of servers with no single point of control. Uses include Instant Messaging (IM), Voice over IP (VoIP) signalling, Internet of Things (IoT) communication, and bridging together existing communication silos - providing the basis of a new open real-time communication ecosystem. While the work of the Spec Core Team is volunteer work, the Matrix.org Foundation spends around $100k US to enable this work through its organisition, by providing infrastructure, and by employing the Director of Standards Development to ensure momentum.",
+        "webpageUrl": {
+            "url": "https://spec.matrix.org",
+            "wellKnown": "https://spec.matrix.org/unstable/.well-known/funding-manifest-urls"
```

Comment:
That is where it currently is. The validator accepts that. The funding.json spec accepts that. The RFC 8615 however requires .well-known to be in the root aiui. The SCT has offered to HTTP302 the root /.well-known to the /unstable/.well-known. I am mailing [info@floss.fund](mailto:info@floss.fund) to try to clarify.

Reply:
If their validator accepts that then I'm keen on merging that and updating it later with a redirect it that's an issue

Reply:
https://github.com/funding-manifest/fundingjson.org/issues/1